azurelinuxagent/common/protocol/hostplugin.py

```diff
@@ -416,18 +423,12 @@ def raise_not_supported(reset_state=False):
             etag = None if force_update or self._cached_vm_settings is None else self._cached_vm_settings.etag
             correlation_id = str(uuid.uuid4())
 
-            def format_message(msg):
-                return "GET vmSettings [correlation ID: {0} eTag: {1}]: {2}".format(correlation_id, etag, msg)
-
-            def get_vm_settings():
```

Comment:
this function was part of the retry logic in a previous iteration, but now the retry is done by the caller so I unfolded this function into the containing code